app/views/layouts/mailer.html.erb

```diff
@@ -3,11 +3,175 @@
+        margin-top: 20px;
+        max-width: 600px;
+        margin-right: auto;
+        margin-left: auto;
+        padding: 20px;
+        border: 1px solid #ddd;
+        border-radius: 15px;
+        box-shadow: 0 7px 30px 0 rgba(33, 55, 74, 0.2)
+      }
+      .mail-wrapper .gravatar {
+        height: 80px;
+        width: 80px;
+        border-radius: 50%;
+        border: 1px solid #ddd;
+      }
+      .mail-wrapper .gravatar img {
+        height: 80px;
+        width: 80px;
+        object-fit: cover;
+        border-radius: 50%;
+      }
+      .mail-wrapper .header {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+      }
+      .mail-wrapper .header.center {
+        justify-content: center;
+      }
+      .mail-wrapper .logo {
+        display: flex;
+        justify-content: center;
+      }
+      .mail-wrapper .logo img {
+        object-fit: cover;
+        height: 80px;
+      }
+      .main {
+        text-align: center;
+      }
+      .main .button {
+        display: inline-block;
+        background-color: #378137; /* #298DCC in pro */
+        border-radius: 5px;
+        padding: 8px 15px;
+        color: #fff;
+        margin: 1rem;
+      }
+      .main .button:hover {
+        background-color: #204b20; /* #2070a2 in pro */
+      }
+      .main .digest-header {
+        font-weight: 300;
+        margin: 1rem 0 1.5rem 0;
+      }
+      .main hr {
+        width: 100%;
+        height: 0;
+        border-top: 1px solid #ddd;
+      }
+      .main .notification {
+        display: flex;
+        align-items: center;
+        margin: 1rem 0;
+      }
+      .main .notification .avatar {
+        height: 50px;
+        width: 50px;
+        border-radius: 50%;
+        border: 1px solid #ddd;
+        margin: 0 20px 0 10px;
+      }
+      .main .notification .avatar img {
+        height: 50px;
+        width: 50px;
+        object-fit: cover;
+        border-radius: 50%;
+      }
+      .main .notification .notification-text {
+        text-align: left;
+        margin: 0;
+        font-weight: 400;
+      }
+      .main .notification .notification-text .user-name {
+        font-weight: 700;
+      }
+      .main .notification .notification-text .time-stamp {
+        font-size: 80%;
+        color: #777;
+        white-space: nowrap;
+      }
+      @media (max-width: 320px) {
+        .main .notification .notification-text .time-stamp {
+          display: block;
+        }
+      }
     </style>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="mail-wrapper">
+
+      <div class="header center">
+        <div class="logo">
+          <%= image_tag 'DradisCE_full_small.png' %> 
+        </div>
+      </div>
+
+      <!-- Dradis Pro Version
+      <div class="header">
+        <div class="logo">
+          <%#= image_tag 'DradisPro_full_small.png' %>
+        </div>
+        <div class="gravatar">
+          <%= image_tag 'profile.jpg' %>
```

Comment:
@MattBudz Who's the gravatar supposed to represent here?

Reply:
@brianp This would be the user that is receiving the email. Thinking about it now, I want to add some logic to only show the gravatar if there is a gravatar set or custom user profile pic uploaded. If it's just going to show the placeholder image, I want it to render just the logo. Ee can make this the same for both CE and Pro since CE now will have working gravatars. The only difference would be the CE vs Pro logo.